src/tools/native-client/pom.xml

```diff
@@ -72,14 +72,12 @@
                             <goal>javah</goal>
                         </goals>
                         <configuration>
-                            <javahOutputDirectory>${basedir}/src/main/cpp
-                            </javahOutputDirectory>
+                            <javahOutputDirectory>${basedir}/src/main/cpp</javahOutputDirectory>
                             <javahOutputFileName>
-                                com_vesoft_client_NativeClient.h
+                                com_vesoft_nebula_codec.h
```

Comment:
the javah output file name should be `package name_file name` ?

Reply:
Ok